src/test/java/webdrivertasks/util/TestListener.java

```diff
@@ -38,7 +38,7 @@ private void saveScreenshot() {
     }
 
     private String getCurrentTimeAsString() {
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("uuuu-MM-dd_HH-mm-ss");
-        return ZonedDateTime.now().format(formatter);
+        final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("uuuu-MM-dd_HH-mm-ss");
```

Comment:
Actually, I meant making the pattern itself a constant, but this one is ok as well